application/client.core/src/app/environment/services/service.file.opener.ts

```diff
@@ -99,13 +98,18 @@ export class FileOpenerService implements IService, IFileOpenerService {
                         // Multiple files
                         // Select way: merge or concat
                         const fileList: FilesList = new FilesList(checkResponse.files);
-                        PopupsService.add({
+                        const guid: string = PopupsService.add({
                             id: 'opening-file-dialog',
                             caption: `Opening files`,
                             component: {
                                 factory: DialogsMultipleFilesActionComponent,
                                 inputs: {
                                     fileList: fileList,
+                                    onEnter: PopupsService.getObservable().onEnter,
+                                    merge: () => {
+                                        this.merge(fileList),
+                                        PopupsService.remove(guid);
+                                    },
```

Comment:
Please rename input `merge` into `defaultAction` or something like it. The name `merge` confuse a little, looks like dialog would be opened to call merge-function